react/react-training/src/app/screens/Dashboard/screens/Profile/layout.js

```diff
@@ -20,9 +21,7 @@ class Layout extends Component {
 
   render() {
     const { userData } = this.props;
-    if (!userData) {
-      return <div className={styles.loader}>Loading...</div>;
-    }
+    if (!userData) return null;
```

Comment:
This is necessary?

Reply:
Yessss... Without this line I'm getting and error. 😿